cves/CVE-2016-10192.yml

```diff
@@ -64,7 +64,15 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A Heap-based buffer overflow vulnerability in ffmpeg's implementation of a 
```

Comment:
This should be an independent clause. Can you change this to something like "There was a heap-based buffer overflow vulnerability in ffmpeg's implementation..."?

Reply:
inserted independent clause